src/index.css

```diff
@@ -33,6 +33,10 @@ code {
   border-bottom: 1px solid rgb(241, 245, 247) !important;
 }
 
+.ui.ribbon.label {
+  left: 0px !important;
```

Comment:
I think this will have an effect if we need the default ribbon else where. Let add a custom className also?